### ISSUE TITLE
Broken soundboard? Fix?

### ISSUE THREAD

**Issue by slyguy65** [jonas747/yagpdb#333]
Failed playing the sound: soundboard.playSound: timeout waiting for voice make sure you put a proper audio file, and did not for example link to a youtube video.

Not sure what they did while I was gone but I came back to the serve giving this message after it had been working for about a week. Now the bot just sits in afk since it doesn't leave voice.

**Comment by jonas747** [comment]
Yeah its been a bit fucky lately, been putting it up since the voice code is a bit spaghetti-ish, probs gonna do a full rewrite
